.github/workflows/ci.yaml

```diff
@@ -8,7 +8,31 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  protolint:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Fetch sources
+        uses: actions/checkout@v3
+        with:
+          submodules: true
+
+      - name: Get protolint
+        run: |
+          # checksum taken from: https://github.com/yoheimuta/protolint/releases/download/v0.38.1/checksums.txt
```

Comment:
Looks like [v0.44.0](https://github.com/yoheimuta/protolint/releases/tag/v0.44.0) is out.

Reply:
Yeah... Looks like the checksums provided for this release are wrong though :facepalm: I will try another version

Reply:
Nevermind, forgot to change a thing